web/modules/custom/dept_migrate/modules/dept_migrate_group_files/config/install/migrate_plus.migration.d7_file_media_video.yml

```diff
@@ -16,7 +16,6 @@ source:
   constants:
     bundle: 'remote_video'
 process:
-  mid: fid
```

Comment:
Got rid because we want to rely on the D9 autoincrement, rather than carry these across verbatim.